tests/src/roomHandlers/fishingRoomHandler.test.ts

```diff
@@ -27,6 +27,112 @@ testSingletonModule(
       testDefaultCommandAtEntrance();
 
       testInvalidCommandAtEntrance();
+
+      testStateTransition(
+        'AtEntrance --> AtEntrance: fish',
+        (): TArrangedTransitionData => ({
+          startingRoomState: {
+            fishCaught: 0,
+            isRodBroken: true,
+            randomNumber: 0,
+          },
+          stateAssertionDescription: 'You can not fish with a broken rod',
+          expectedResult: {
+            commandDescription:
+              'You sit down and start to get ready to fish. But your rod is broken and you have to fix it.',
+            roomState: {
+              fishCaught: 0,
+              isRodBroken: true,
+              randomNumber: 0,
+            },
+          },
+        }),
+      );
+
+      testStateTransition(
+        'AtEntrance --> Fishing: fish',
+        (): TArrangedTransitionData => ({
+          startingRoomState: {
+            fishCaught: 0,
+            isRodBroken: false,
+            randomNumber: 0,
+          },
+          stateAssertionDescription: null,
+          expectedResult: {
+            commandDescription:
+              'You sit down and start to get ready to fish. This is going to be alot of fun!',
+            roomState: {
+              fishCaught: 0,
+              isRodBroken: false,
+              randomNumber: 0,
+            },
+          },
+        }),
+      );
+
```

Comment:
You're missing `AtEntrance --> Fix` in both your markdown file and in the tests

Reply:
Oh yeah.... probably because it was missing from the docs. Updated.